index.html

```diff
@@ -0,0 +1,94 @@
+         <div class="container1">
+            <div class="item1">css</div>
+            <div class="item2">html</div>
+            <div class="item3">bootstrap</div>  
+            <div class="item2">Ruby</div>  
+          </div>
+          <button class="button">See Projects</button>
+        </div>
+        <div class="container2">
+            <div class="Data"><h1>Profesional Art Printing Data</h1></div>
+         <div class="para1"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p></div>
+         <div class="container3">
+            <div class="item4">css</div>
+            <div class="item5">bootstrap</div>
+            <div class="item6">Ruby</div>
+         </div>
+        </div>
+        <div class="container4">
+            <button class="button1">See Projects</button>
+        </div>
+        </section>
+        <section class="about me">
+            <div class="container5">
+                <div class="about"><h1>About me</h1></div>
+                <div class="me"><p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+            </div>
+            <div class="container6">
+                <button class="button2">Get My Resume</button>
+                <div class="image"><img src="./image/Frame.png" alt=""></div>
+            </div>
+            <div class="container7">
+                <div class="vector4"><img src="./image/Vector 4.png" alt=""></div>
+                <div class="languages">
+                    <div class="55"><img src="./image/Rectangle 55.png" alt=""></div>
+                    <div class="lang"><h2>Language</h2></div>
+                </div>
+                <div class="points">
+                <div class="java"><p>Javascript</p></div>
+                <div class="ruby"><p>Ruby</p></div>
+                <div class="html"><p>Html</p></div>
+                <div class="css"><p>Css</p></div>
+                </div>
+                
+            </div>
+            <div class="container8">
+                <div class="languages">
+                    <div class="55"><img src="./image/Rectangle 56.png" alt=""></div>
+                    <div class="lang"><h2>Frameworks</h2></div>
+                </div>
+                <div class="points">
+                <div class="java"><p>Bootstrap</p></div>
+                <div class="ruby"><p>Ruby on Rails</p></div>
+                <div class="html"><p>RSPec</p></div>
+                <div class="css"><p>CapyBara</p></div>
+                <div class="css"><p>Selenium</p></div>
+                </div>
```

Comment:
- [ ] The same case is in your about me section. Kindly use `<ul>` and `<li>` tags to group and list your technologies